lib/src/top_level.rs

```diff
@@ -561,8 +609,17 @@ fn fn_to_doc<'a>(
     args: &'a Vec<(String, CoqType)>,
     ret_ty: &'a Option<CoqType>,
     body: &'a Expr,
+    is_dead_code: bool,
 ) -> Doc<'a> {
-    nest([
+    group([
+        if is_dead_code {
+            concat([
+                text("(* #[allow(dead_code)] - function was ignored by the compiler *)"),
+                hardline(),
+            ])
+        } else {
+            nil()
+        },
         nest([
```

Comment:
With one more `nest` to indent the body of the definitions (closed at the end of this printing function)
```suggestion
        nest([
        nest([
```